### ISSUE TITLE
LSP_train.py - Training stuck at 0/num_optimum_steps

### ISSUE THREAD

**Issue by Mayanksoni20** [microsoft/DialoGPT#49]
**Issue :**  The pbar for training is stuck at 0/num_optim_steps. Could be seen in the image below.
 
When i clone the repository and run `python demo.py` everything works find until the training loop. Then the pbar is stuck at 0/num_optim_steps meaning that the training does not take place. By default `python demo.py` takes `--data = dummy_data`. 

**Steps taken towards resolution**

- Manually ran the command generated such as `python LSP_train.py \
--model_name_or_path /home/msoni/DialoGPT/models/small \
--init_checkpoint /home/msoni/DialoGPT/models/small/pytorch_model.bin \
--train_input_file /home/msoni/DialoGPT/data/train.128len.db \
--eval_input_file ./data/dummy_data.tsv \
--output_dir /home/msoni//DialoGPT/models/output_model \
--seed 42 \
--max_seq_length 128 \
--train_batch_size 8 \
--gradient_accumulation_steps 8 \
--eval_batch_size 64 \
--learning_rate 1e-05 \
--num_optim_steps 1000 \
--valid_step 500 \
--warmup_steps 100 \
--normalize_data true \
--fp16 False \
--lr_schedule noam \
--loss_scale 0.0 \
--no_token_id true \
--pbar true \
--local_rank -1`

- Altered the parameters for num_optim_steps, valid_steps, and warmup_steps.

- Turned from_scratch = True

- After the while loop at line 273 in LSP_train.py, printed a statement if that line executes. The code skips all lines after line number 290. This is the place the skip starts

`if args.no_token_id:
            token_ids = None
        loss, ppl = model(input_ids, position_ids, token_ids, label_ids)
        logger.info("loss", loss)
        logger.info("ppl", ppl)
        logger.info("if this command print than line number 294 has run")`

Is there something obvious that i am missing ?
![Error](https://user-images.githubusercontent.com/7255078/92749953-8457d900-f37e-11ea-95dd-008f72bfa13b.JPG)

**Comment by haozhe-an** [comment]
Hi, I'm having the same issue. Did you perhaps figure out how to solve it? Thanks!

**Comment by Zoetic-Zephyr** [comment]
Same issue here. Also happens when training with more than one GPU. Would it possible to share how to fix the progress bar? Thank you.
